ExcelToCWMS/ConfigReader.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ExcelToCWMS
+{
+    public class ConfigReader
+    {
+        private string configFile;
+        private Dictionary<string, string> dict;
+        public ConfigReader(string configFile)
+        {
+            var lines = File.ReadAllLines(configFile);
+            Dictionary<string, string> dict = new Dictionary<string, string>();
```

Comment:
it may be cleaner to use the member variable dict here, instead of a locally scoped  dict with the same name.  Or try a convention such as m_dict; indicating member variable to distinguish between this.dict.

Reply:
trashed the local dict and am now only using member variable  m_Dict